src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -234,6 +241,37 @@ const Actions: React.FC<Props> = ({
         };
     };
 
+    const mapsLinkForParcels = (): string => {
+        let mapsLinkString = "https://www.google.com/maps/dir";
+        selectedParcels.forEach((parcel) => {
+            mapsLinkString = mapsLinkString.concat("/", parcel.addressPostcode);
+        });
+        mapsLinkString = mapsLinkString.concat("//");
+        return mapsLinkString;
+    };
+
+    const openInNewTab = (url: string): void => {
+        const newWindow = window.open(url, "_blank", "noopener, noreferrer");
+        if (newWindow) {
+            newWindow.opener = null;
```

Comment:
What does this do? Looks like the line above guarantees the opener is set to null anyway? (though i'm not sure how this window.open business works)